app/controllers/mission_control/jobs/application_controller.rb

```diff
@@ -4,7 +4,15 @@ class MissionControl::Jobs::ApplicationController < MissionControl::Jobs.base_co
   include MissionControl::Jobs::ApplicationScoped, MissionControl::Jobs::NotFoundRedirections
   include MissionControl::Jobs::AdapterFeatures
 
+  before_action :http_auth
```

Comment:
I'd rename this to `authenticate_by_basic_auth` and then make it conditional on another property `http_auth_enabled` (similar to `http_auth_user`), which would default to `true`, just to disable it easily.

Reply:
Will take care on it